src/util/pg.py

```diff
@@ -112,6 +117,20 @@ def execute(query):
             )
 
 
+def format(cr, query, *args, **kwargs):
```

Comment:
As shown by the example in the docstring, `format` is too generic.
```suggestion
def format_query(cr, query, *args, **kwargs):
```